app/server.go

```diff
@@ -74,13 +86,13 @@ func (s *server) middleware(n httprouter.Handle) httprouter.Handle {
 			"content-length": r.ContentLength,
 		}).Debugf("HTTP Request to %s", r.URL)
 
-		if r.ContentLength > 0 {
-			// Dump payload into logs for visibility
-			b, err := ioutil.ReadAll(r.Body)
-			if err == nil {
-				log.Debugf("Dumping Payload for request to %s: %s", r.URL, b)
```

Comment:
I'd like to keep the logs consistently using the WithFields above. It will help keep context when debugging.